src/super_gradients/training/models/detection_models/csp_darknet53.py

```diff
@@ -131,6 +131,48 @@ def forward(self, x):
         return self.cv3(torch.cat((self.m(self.cv1(x)), self.cv2(x)), dim=1))
 
 
+class CSPLayer(nn.Module):
+    """C3 in yolov5, CSP Bottleneck with 3 convolutions"""
+
+    def __init__(
+        self,
+        in_channels,
+        out_channels,
+        n=1,
+        act=nn.SiLU,
+        shortcut=True,
+        depthwise=False,
+        expansion=0.5,
+
+    ):
+        """
+        Args:
+            in_channels (int): input channels.
+            out_channels (int): output channels.
+            n (int): number of Bottlenecks. Default value: 1.
+        """
+        # ch_in, ch_out, number, shortcut, groups, expansion
+        super().__init__()
+        hidden_channels = int(out_channels * expansion)  # hidden channels
```

Comment:
Not that it matters much, but very informative comment out there 
`hidden_channels = ...  # hidden channels`
:D